File: homework/testing/h10/h10t.rb
require_relative "h10"
require "test/unit"

RESULT = summingPairs(INPUT,2020)

class SimpleTests < Test::Unit::TestCase
  def test_contains_checks
    assert_equal(true, RESULT.include?(Pair.new(374, 150)))
    assert_equal(true, RESULT.include?(Pair.new(626, 1030)))
    assert_equal(true, RESULT.include?(Pair.new(150, 1850)))
  end
end

File: homework/testing/h10/h10t.clj
(ns testing)
(use 'clojure.test)
(load-file "collection.clj") ;; The `collection` file provided with the homework.
(load-file "h10.clj")

(deftest contains-checks
  (let [result (summingPairs input 2020)]
    (is (contains? (set result) [374 150]))
    (is (contains? (set result) [626 1030]))
    (is (contains? (set result) [150 1850]))))




(deftest test-ns-hook
  (contains-checks))

(run-tests 'testing)
